gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -5170,5 +5180,41 @@ subroutine RecordRestart( gc, import, export, clock, rc )
     _RETURN(ESMF_SUCCESS)
   end subroutine RecordRestart
 
+  subroutine  checkIfStateHasField(state, fieldName, hasField, rc)
```

Comment:
No change requested,  but in the future consider that a LOGICAL function would have expressed the functionality more cleanly.